client/package.json

```diff
@@ -27,32 +27,41 @@
     "singleQuote": true
   },
   "dependencies": {
+    "@apollo/client": "^3.7.10",
     "@emotion/react": "^11.10.0",
     "@emotion/styled": "^11.10.0",
     "@fontsource/roboto": "^4.5.8",
     "@mui/icons-material": "^5.8.4",
     "@mui/material": "^5.10.0",
+    "axios": "^1.3.4",
+    "graphql": "^16.6.0",
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.3.0",
     "recharts": "^2.1.13",
     "resize-observer-polyfill": "^1.5.1",
     "serve": "^14.0.1",
-    "typescript": "^4.9.5"
+    "typescript": "^4.7.4"
   },
   "devDependencies": {
+    "@babel/cli": "^7.18.10",
     "@babel/core": "^7.21.3",
+    "@babel/eslint-parser": "^7.18.9",
```

Comment:
@SyntaxXeror , can you confirm that all these new dependencies are needed for this PR?

Reply:
We have double checked and these dependencies were from old commits of auth and are not required any longer, we will update the package.json in the next commit.